src/components/RegisterPage/RegisterPage.jsx

```diff
@@ -0,0 +1,175 @@
+                                <Form.Control
+                                    type="text"
+                                    required
+                                    placeholder="John"
+                                    value={data.name}
+                                    onChange={e => setData({...data, name: e.target.value})}
+                                />
+                                <Form.Control.Feedback type="invalid">
+                                    First name must be provided
+                                </Form.Control.Feedback>
+                            </Form.Group>
+                        </Col>
+                        <Col>
+                            <Form.Group controlId="lastname">
+                                <Form.Label>Last name:</Form.Label>
+                                <Form.Control
+                                    type="text"
+                                    placeholder="Doe"
+                                    required
+                                    value={data.surname}
+                                    onChange={e => setData({...data, surname: e.target.value})}
+                                />
+                                <Form.Control.Feedback type="invalid">
+                                    Last name must be provided
+                                </Form.Control.Feedback>
+                            </Form.Group>
+                        </Col>
+                    </Row>
+                    <Form.Group controlId="email">
+                        <Form.Label>Email:</Form.Label>
+                        <Form.Control
+                            type="email"
+                            required
+                            placeholder="johndoe@example.com"
+                            value={data.email}
+                            onChange={e => setData({...data, email: e.target.value})}
+                        />
+                        <Form.Control.Feedback type="invalid">
+                            Correct email must be provided
+                        </Form.Control.Feedback>
+                    </Form.Group>
+                    <Form.Group controlId="password">
+                        <Form.Label>Password:</Form.Label>
+                        <Form.Control
+                            type="password"
+                            minLength={8}
+                            required
+                            isInvalid={data.password && data.password_R && data.password !== data.password_R}
+                            placeholder="Your password"
+                            name="password"
+                            value={data.password}
+                            onChange={handlePassword}
+                        />
+                        <Form.Control.Feedback type="invalid">
+                            Password must be at least 8 characters long, contain one capital, one number and one special character
```

Comment:
Nie wiem czy jest sens wspominać o tych capital letters i numbers skoro tego nie robimy, bo takie warunki aż proszą się żeby je przetestować, a jak mamy tylko wymóg 8 liter to jeden teścik i z głowy